libs/vm-modules/src/math/tensor.cpp

```diff
@@ -57,19 +57,20 @@ Ptr<VMTensor> VMTensor::Constructor(VM *vm, TypeId type_id, Ptr<Array<SizeType>>
 
 void VMTensor::Bind(Module &module)
 {
+  using Index = fetch::math::SizeType;
   module.CreateClassType<VMTensor>("Tensor")
       .CreateConstructor(&VMTensor::Constructor)
       .CreateSerializeDefaultConstructor([](VM *vm, TypeId type_id) -> Ptr<VMTensor> {
         return Ptr<VMTensor>{new VMTensor(vm, type_id)};
       })
-      .CreateMemberFunction("at", &VMTensor::AtOne)
-      .CreateMemberFunction("at", &VMTensor::AtTwo)
-      .CreateMemberFunction("at", &VMTensor::AtThree)
-      .CreateMemberFunction("at", &VMTensor::AtFour)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtOne)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtTwo)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtThree)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtFour)
+      .CreateMemberFunction("at", &VMTensor::At<Index>)
```

Comment:
it's annoying but there's no obvious way around this right now is there?
If so, would you mind please adding in the bindings for the 5 & 6 dimensional tensor accessor/setter as well. There are some cases with convolution where you might get such a tensor.

Reply:
you can do a separate pr for that if you like

Reply:
Done